doc/DeveloperGuide.md

```diff
@@ -94,6 +94,9 @@ Use case ends.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
+5. Should store record in human readable files.
```

Comment:
What do you mean human readable files? It seems general... perhaps `Should store record in english and spanish unicode encoding`.